<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create</class>
 <widget class="QMainWindow" name="create">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>507</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>391</width>
    <height>507</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>391</width>
    <height>507</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create an account </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="edit_username">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MathJax_Caligraphic&quot;;
color:#4e4e4e;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MathJax_Caligraphic&quot;;
color:#4e4e4e;
</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_password">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MathJax_Caligraphic&quot;;
color:#4e4e4e;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_confirm_password">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MathJax_Caligraphic&quot;;
color:#4e4e4e;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>46</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MathJax_Caligraphic&quot;;
color:#4e4e4e;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MathJax_Caligraphic&quot;;
color:#4e4e4e;</string>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MathJax_Caligraphic&quot;;
color:#4e4e4e;</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MathJax_Caligraphic&quot;;
color:#4e4e4e;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_account_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MathJax_Caligraphic&quot;;
color:#4e4e4e;</string>
    </property>
    <property name="text">
     <string>  Create </string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>:/src/link.png</selectedon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="go_to_login_link">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MathJax_Caligraphic&quot;;
color:#4e4e4e;</string>
    </property>
    <property name="text">
     <string>You already have an account?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
